### PR TITLE
SY-930 add console tests to ci

### PR DESCRIPTION
**Pull request by Lham42**
# Feature Pull Request Template

## Key Information

- **Linear Issue**: [SY-930](https://linear.app/synnaxlabs/issue/SY-930/[console]-add-console-tests-to-release-ci)

## Description

Added console tests to CI

## Basic Readiness Checklist

- [x] I have performed a self-review of my code.
- [x] I have added relevant tests to cover the changes to CI.
- [x] I have updated user facing documentation accordingly.
- [x] I have verified code coverage targets are met.

## Migrations

- [ ] Console - I have ensured that previous versions of stored data structures are 
properly migrated to new formats.
- [ ] Server - I have ensured that previous versions of stored data structures are 
properly migrated to new formats.

##  Additional Notes
- [ ] These changes deal with concurrency.
- [ ] These changes affect UI.

## Manual QA Additions

- [ ] I have updated the [Release Candidate](/.github/PULL_REQUEST_TEMPLATE/rc.md) template
with necessary manual QA steps to test my change.

## Breaking Changes

Please list any breaking changes to public or internal packages.

## Reviwer Checklist
- [ ] Sufficient test coverage of new additions.
- [ ] Verified all steps in the Readiness checklists.
- [ ] UI changes have been tested.
- [ ] Style and formatting is consistent.
- [ ] Reviewed any relevant changes to concurrent code for safety. 
- [ ] Sufficient comments and clarity of code.
- [ ] Any additional documentation necessary for feature is provided and has been reviewed for clarity.

### CODE REVIEW

File: console/package.json [hunk 1]
```diff
@@ -49,6 +50,7 @@
     "@types/uuid": "^10.0.0",
     "@vitejs/plugin-react": "^4.3.1",
     "eslint-config-synnaxlabs": "workspace:*",
+    "jest": "^29.7.0",
```
Comment (pjdotson) [inline console/package.json:53]: Get rid of this dependency and run `pmpn install` again in root directory.

**Review by pjdotson — APPROVED** [review]
Approved once comment is addressed.

File: .github/workflows/test.console.yaml
Comment (emilbon99) [inline .github/workflows/test.console.yaml:1]: This workflow doesn't require us to build the console. You can get rid of most of these statements. I'll mark the ones you can get rid of.

**Review by emilbon99 — CHANGES_REQUESTED** [review]

**Review by emilbon99 — APPROVED** [review]
LGTM

**Pull request merged**